src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -14,7 +14,7 @@
 public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
-    private Path addressBookFilePath = Paths.get("data" , "addressbook.json");
+    private Path addressBookFilePath = Paths.get("data" , "stonksBook.json");
```

Comment:
Based on your screenshots, this should be lowercase right? 

```suggestion
    private Path addressBookFilePath = Paths.get("data" , "stonksbook.json");
```

Reply:
Hmmmm the original was addressBook.json here so I kept it in camel case. I will try changing to lower case and see if anything goes wrong.